composer.json

```diff
@@ -135,8 +138,8 @@
         "drupal/responsive_preview": "^1.0",
         "drupal/redirect": "^1.0",
         "drupal/riddle_marketplace": "^2.0",
-        "drupal/scheduler": "1.0",
-        "drupal/scheduler_content_moderation_integration": "~1.0",
+        "drupal/scheduler": "^1.0",
+        "drupal/scheduler_content_moderation_integration": "dev-task/add-scheduler-patch",
```

Comment:
the module has a release now, we can go back to require with the version constraint.